Orange/widgets/classify/tests/test_owadaboost.py

```diff
@@ -0,0 +1,53 @@
+from Orange.widgets.tests.base import GuiTest
+from Orange.widgets.classify.owadaboost import OWAdaBoostClassification
+from PyQt4 import QtGui
+
+
+class TestOwadaboost(GuiTest):
```

Comment:
Test class name should include full widget name:
  TestOWAdaBoostClassification